github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml

```diff
@@ -223,6 +223,10 @@ tide:
         repos:
           kubernetes-nmstate:
             skip-unknown-contexts: true
+  priority:
+  - labels: [ "kind/flake", "priority/critical-urgent" ]
+  - labels: [ "kind/failing-test", "priority/critical-urgent" ]
+  - labels: [ "kind/bug", "priority/critical-urgent" ]
```

Comment:
For my understanding, I guess we will need the combination of these labels in order for the priority to kick in?

Reply:
Yes, exactly. I was just thinking that we test this beforehand - i.e. before we announce the feature/